api/services/userService.js

```diff
@@ -0,0 +1,50 @@
+const bcrypt = require('bcrypt')
+
+const { userDao } = require('../models')
+
+const hashPassword = async (plainTextPassword) => {
+    try {
+        const SALT_ROUNDS = 12;
+        const salt = await bcrypt.genSalt(SALT_ROUNDS);
+        const hashedPassword = await bcrypt.hash(plainTextPassword, salt);
+        return hashedPassword;
+    } catch {
+        const error = new Error('INVALID_HASH')
+        error.statusCode = 400
+        throw error
+    }
```

Comment:
암호화 하는 부분에 있어 발생하는 에러에 대한 예외처리를 하는 것도 좋지만, 제가 보기엔 없어도 될 것 같습니다!
```suggestion
        const SALT_ROUNDS = 12;
        const salt = await bcrypt.genSalt(SALT_ROUNDS);
        const hashedPassword = await bcrypt.hash(plainTextPassword, salt);
        
        return hashedPassword;
```